src/interaction/drawazimut.js

```diff
@@ -87,8 +88,7 @@ ngeo.interaction.DrawAzimut = function(options) {
       useSpatialIndex: false,
       wrapX: false
     }),
-    style: options.style !== undefined ?
-      options.style : ol.interaction.Draw.getDefaultStyleFunction()
+    style: options.style !== undefined ? options.style : ngeo.interaction.getDefaultDrawStyleFunction()
```

Comment:
You could simplify to
`style: options.style || ngeo.interaction.getDefaultDrawStyleFunction()`.